pkg/brew/bfs.rb

```diff
@@ -0,0 +1,18 @@
+class Bfs < Formula
+  version '1.0.1'
```

Comment:
From what I know of brew, there is no way to provide a tap to a ref.  Besides, you wouldn't *really* want to do that because it would require users to 're-tap' every update instead of just doing `brew upgrade bfs`.